README.md

```diff
@@ -274,15 +272,15 @@ To give a cost comparison with retail pricing in the East US region with 1 TB of
 
 ### Cost Increase
 
-Using default settings, each instance of AzureSharedResource will make a single storage transaction roughly every 250 milliseconds _when it needs additional capacity_. Therefore, we can determine the maximum cost for 4 instances on an Azure Storage Account GPv2 (or Blob) as...
+Using default settings, each instance of SharedResource will make a single storage transaction roughly every 250 milliseconds _when it needs additional capacity_. Therefore, we can determine the maximum cost for 4 instances on an Azure Storage Account GPv2 (or Blob) as...
```

Comment:
This should be referring to the lease manager for Azure, not the generic SharedResource that might only be configured for Reserved capacity

Reply:
Adjusted verbiage. Thanks @KeithJRome